src/Symfony/Component/Console/DependencyInjection/AddConsoleCommandPass.php

```diff
@@ -35,22 +48,41 @@ public function process(ContainerBuilder $container)
             if (!$r = $container->getReflectionClass($class)) {
```

Comment:
it can't happen, this already throws on abstracts https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/DependencyInjection/AddConsoleCommandPass.php#L28